zerver/tests/test_custom_profile_data.py

```diff
@@ -3,23 +3,23 @@
 from typing import Union, List, Dict, Text, Any
 from mock import patch
 
-from zerver.lib.actions import get_realm, try_add_realm_custom_profile_field, \
+from zerver.lib.actions import try_add_realm_custom_profile_field, \
     do_update_user_custom_profile_data, do_remove_realm_custom_profile_field
 from zerver.lib.test_classes import ZulipTestCase
 from zerver.models import CustomProfileField, \
-    custom_profile_fields_for_realm, get_user
+    custom_profile_fields_for_realm, get_realm
 import ujson
 
+FIELDS_ALREADY_EXISTS = len(custom_profile_fields_for_realm(get_realm("zulip").id))
```

Comment:
As a general rule in a large application, one should never do database queries outside a function (because that code runs at import time, not at runtime, which can be really confusing and cause initialization problems)

I'm just going to add a new commit refactoring that to live in a `setUp` method instead, which is the intended place for this sort of thing in a unit test.

Reply:
(I think even better would be to redo these tests to not be sensitive to how many fields there were before, and instead just look at what happened with the fields they are trying to interact with...)

Reply:
Yeah, this sounds more clean and correct. Will cover this in next version of this PR.